broker/src/main/java/io/zeebe/broker/transport/commandapi/CommandApiMessageHandler.java

```diff
@@ -49,14 +51,18 @@
   protected final Consumer<Runnable> cmdConsumer = Runnable::run;
 
   protected final Int2ObjectHashMap<LogStream> leadingStreams = new Int2ObjectHashMap<>();
+  protected final Int2ObjectHashMap<RequestLimiter<Void>> partitionLimiters =
```

Comment:
Why not just use directly the `PartitionAwareRequestLimiter` here? Is there a reason we track partitions and their limiters in more than one place?